spec/PreviewEntry.test.js

```diff
@@ -4,40 +4,34 @@ import renderer from 'react-test-renderer';
 import 'jest-enzyme';
 import 'jest-styled-components';
 import { mount } from 'enzyme';
-import styled from 'styled-components';
 
-import PreviewEntry from '../client/src/components/PreviewEntry.jsx';
+import PreviewEntry from '../client/src/components/PreviewEntry.jsx'; //eslint-disable-line
```

Comment:
I think it'd be nicer to just get rid of this whitespace instead of ignoring eslint assuming that's the issue. Or maybe eslint doesn't like the '.jsx', you can configure webpack to add that for you, but I wouldn't worry about it right now.